### PR TITLE
feat(geography): shard packages

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## What is the new behavior?
- Splits packages up into subpackages
- Adds a `tsconfig` that maps geography packages to root, useful for testing
- Changes some test imports to reference the `@daffodil` package.
- Changes `tsconfig.json` to use a wildcard import for `@daffodil/*`

## Does this PR introduce a breaking change?
```
[x] Yes
[ ] No
```

Many imports have moved. `DaffGeographyModule` has been removed.
`DaffStateModule` must be manually imported. It is exported by `@daffodil/geography/state`.

## Other information

### CODE REVIEW

File: libs/geography/src/geography.module.ts [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
   imports: [
     CommonModule,
-
-    /**
-     * Ngrx/store
-     */
-    DaffGeographyStateModule,
   ]
 })
 export class DaffGeographyModule {}
```
Comment (lderrickable) [inline libs/geography/src/geography.module.ts:9]: Should we even have this file anymore, since it doesn't export anything?

Comment (griest024) [reply libs/geography/src/geography.module.ts:9]: Yeah I guess it's actually useless. I'll remove it.

Comment (griest024) [reply libs/geography/src/geography.module.ts]: removed

**Comment by griest024** [comment]
\azp run

File: libs/geography/driver/testing/in-memory/package.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "name": "@daffodil/geography/driver/testing/in-memory"
```
Comment (damienwebdev) [inline libs/geography/driver/testing/in-memory/package.json:2]: I think in-memory is not a part of the "testing" package? I think it's worthy of its own driver unrelated to "testing".

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: libs/geography/driver/testing/in-memory/package.json (hunk 2 above)
Comment (griest024) [reply libs/geography/driver/testing/in-memory/package.json]: yeah that makes sense. changed

**Review by damienwebdev — APPROVED** [review]

**Pull request merged**